.sbtopts-nightly-sigar

```diff
@@ -0,0 +1,19 @@
+-J-XX:+UseG1GC -Xms3g -Xmx3g 
+-J-XX:-UseBiasedLocking 
+-J-XX:+UseCompressedOops 
+-J-XX:+PrintGCDetails 
+-J-XX:+PrintGCTimeStamps 
+-J-XX:MaxPermSize=512M 
+-J-XX:MetaspaceSize=512M 
+-J-Xss2m 
+-J-XX:ReservedCodeCacheSize=256m 
+-Djava.security.egd=file:/dev/./urandom
+-Dakka.test.sigar=true 
```

Comment:
Ah, perhaps the regular tests are ran without this option?

Reply:
yes.

Reply:
I have no idea how relevant the specific test for sigar is TBH.